client/src/landing/SectionShowcase.tsx

```diff
@@ -0,0 +1,230 @@
+    height: isHeightFixed ? `${heightRem}rem` : undefined,
+    lineClamp: isHeightFixed ? numberLines : undefined,
+    margin: "12px 0 0 0",
+    minHeight: isHeightFixed ? `${heightRem}rem` : undefined,
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    WebkitBoxOrient: "vertical",
+    WebkitLineClamp: isHeightFixed ? numberLines : undefined,
+  };
+
+  return (
+    <div className={cx(className)} style={style}>
+      {children}
+    </div>
+  );
+}
+
+function ShowcaseCardDisplay({
+  description,
+  identifier,
+  imageUrl,
+  title,
+}: ShowcaseCardDisplayProps) {
+  const hasImage = imageUrl != null;
+  const imageStyles = hasImage
+    ? {
+        backgroundImage: `url("${imageUrl}")`,
+        backgroundPositionX: "center",
+        backgroundSize: "contain",
+        backgroundRepeat: "no-repeat",
+      }
+    : {};
+  return (
+    <div
+      data-cy="showcase-card"
+      className={cx(
+        "col text-decoration-none mb-5 border",
+        styles.sectionShowcaseCard
+      )}
+    >
+      <div className="card">
+        <div
+          style={imageStyles}
+          className={cx(
+            styles.cardHeaderEntity,
+            !hasImage && styles.cardHeaderEntity__missing
+          )}
+        >
+          {!hasImage && (
+            <div className="card-bg-title user-select-none">Missing Image</div>
+          )}
+        </div>
+        <div className={styles.cardBody}>
+          <FixedLineHeightContainer
+            isHeightFixed={true}
+            lineHeightRem={1.9}
+            numberLines={3}
+          >
+            <h4 className="lh-sm">{title}</h4>
+          </FixedLineHeightContainer>
+          <FixedLineHeightContainer isHeightFixed={true} numberLines={5}>
+            <RenkuMarkdown markdownText={description} />
+          </FixedLineHeightContainer>
+          <div className="mt-4 mb-4">
+            <Link
+              to={`/projects/${identifier}`}
+              className="btn btn-sm btn-secondary"
+            >
+              Open
+            </Link>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+type ShowcaseCardProps = {
+  identifier: string;
+  overrideDescription?: string;
+  overrideImageUrl?: string;
+  overrideTitle?: string;
+};
+function ShowcaseCard({
+  identifier,
+  overrideDescription,
+  overrideImageUrl,
+  overrideTitle,
+}: ShowcaseCardProps) {
+  const { data: projectMetadata, isLoading } = useProjectMetadataQuery({
+    projectPath: identifier,
+  });
+  if (isLoading) return <Loader />;
+  const images = projectMetadata?.images;
+  const details: Partial<ShowcaseCardDisplayProps> = {
+    description: projectMetadata?.description,
+    identifier,
+    imageUrl: images?.length ? images[0].location : undefined,
+    title: projectMetadata?.name,
+  };
+  if (overrideDescription) details.description = overrideDescription;
+  if (overrideImageUrl) details.imageUrl = overrideImageUrl;
+  if (overrideTitle) details.title = overrideTitle;
```

Comment:
This is simpler to understand:

```suggestion
  const details: Partial<ShowcaseCardDisplayProps> = {
    description: overrideDescription ?? projectMetadata?.description,
    identifier,
    imageUrl: overrideImageUrl ?? images?.at(0)?.location,
    title: overrideTitle ?? projectMetadata?.name,
  };
```